src/EditorFeatures/Core/Tagging/TaggerContext.cs

```diff
@@ -23,7 +24,7 @@ internal class TaggerContext<TTag> where TTag : ITag
         private readonly ImmutableDictionary<ITextBuffer, TagSpanIntervalTree<TTag>> _existingTags;
 
         internal ImmutableArray<SnapshotSpan> _spansTagged;
-        internal ImmutableArray<ITagSpan<TTag>>.Builder tagSpans = ImmutableArray.CreateBuilder<ITagSpan<TTag>>();
+        public readonly SegmentedList<ITagSpan<TTag>> TagSpans = new();
```

Comment:
>public readonly SegmentedList<ITagSpan<TTag>> TagSpans = new(); [](http://example.com/codeflow?start=8&length=63)

Intentional decision to expose a mutable object from here, right?

Reply:
correct.  (this was already exposed and already mutable).